### ISSUE TITLE
Add support for auto discovery of Amazon Elasticache cluster

### ISSUE THREAD

**Issue by snowtorious** [alexzel/iomem#27]
Amazon Elasticache provides an "auto discovery" mechanism which allows client programs to automatically identify all of the nodes in a cache cluster, and to initiate and maintain connections to all of these nodes. Some other client libraries support this. This feature would be a useful addition to iomem.

**Additional context**
[Adding Auto Discovery To Your Client Library](https://docs.aws.amazon.com/AmazonElastiCache/latest/mem-ug/AutoDiscovery.AddingToYourClientLibrary.html)

**Comment by alexzel** [comment]
Thank you for the feature suggestion. It seems like not a part of the standard Memcached protocol. I wonder if it can be implemented as a wrapper around this library. If you strongly feel that it has to be here, I would really appreciate any help with the code changes. For now I'm adding this request into v2.0.0 [discussion](https://github.com/alexzel/iomem/discussions/20), so we can revisit it later.

**Comment by snowtorious** [comment]
Yes, I think it could be implemented as a wrapper. I attempted to do this by sending the `config get` command like so:

```
client._net.query(['config', 'get', 'cluster'])
```

However, this appears to fail due to `config get` not being defined in protocol.js. This makes sense as iomem uses the binary protocol. Perhaps a list of additional opcodes could be provided to the client? (Opcode for config get command is `0x60`.)

**Comment by alexzel** [comment]
yes, you need to change it in some places (actually a lot) https://gist.github.com/alexzel/cfbfcd79d9816253cbca48b7b4404baa
